### ISSUE TITLE
SciPy 1.3.0 Issue with "comb"

### ISSUE THREAD

**Issue by mbeach42** [weinbe58/QuSpin#230]
Looks like there may be an issue install quspin with scipy 1.3.0. They may have changed how "comb" is exported, i.e. `from scipy.special import comb`. scipy 1.2.2 works fine for now. 

```
Python 3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 21:52:21) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import quspin
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/__init__.py", line 3, in <module>
    from . import operators
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/operators/__init__.py", line 52, in <module>
    from .quantum_operator_core import *
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/operators/quantum_operator_core.py", line 3, in <module>
    from ..basis import spin_basis_1d as _default_basis
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/basis/__init__.py", line 108, in <module>
    from .basis_1d import *
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/basis/basis_1d/__init__.py", line 1, in <module>
    from .spin import spin_basis_1d
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/basis/basis_1d/spin.py", line 1, in <module>
    from ._basis_1d_core import hcp_basis,hcp_ops
  File "/home/meach/anaconda3/envs/quspin/lib/python3.7/site-packages/quspin/basis/basis_1d/_basis_1d_core/__init__.py", line 1, in <module>
    from . import hcp_basis,hcp_ops
  File "quspin/basis/basis_1d/_basis_1d_core/hcp_basis.pyx", line 17, in init quspin.basis.basis_1d._basis_1d_core.hcp_basis
ImportError: cannot import name comb
```

For reference, this is on Linux, 

```active environment : quspin
    active env location : /home/meach/anaconda3/envs/quspin
            shell level : 2
       user config file : /home/meach/.condarc
 populated config files : /home/meach/.condarc
          conda version : 4.7.5
    conda-build version : 3.18.7
         python version : 3.7.3.final.0
               platform : linux-64
             user-agent : conda/4.7.5 requests/2.22.0 CPython/3.7.3 Linux/5.2.0-arch2-1-ARCH 
```

**Comment by mgbukov** [comment]
yes, this is an issue induced by a scipy update. We will fix it with the next release. Until then you can try to downgrade you scipy version; the following should be fine:
```
conda install scipy=1.2.1
```

**Comment by mgbukov** [comment]
We need to replace the code by

```
try:
    from scipy.misc import comb
except ImportError:
    from scipy.special import comb
```

**Comment by weinbe58** [comment]
fixed in #235

**Comment by mgbukov** [comment]
reopening this until we do the official release

**Comment by mgbukov** [comment]
here're more info on how to deal with this issue of downgrading scipy messes up other package dependences (courtesy by to W-W Ho)

The latest version of scipy=1.3.0 does not support the comb functionality.

One has to revert back to a earlier version of scipy, e.g. scipy=1.2.1

Perform these actions. In terminal, create an environment
`
conda create --name myenv
`
where `myenv`  is your choice of the environment name.

Then install scipy in it,
`
conda install -n myenv scipy=1.2.1
`
as well as quspin
`
conda install -n myenv -c weinbe58 quspin
`
Install `ipykernel` as well
`
conda install -n myenv ipykernel
`
Activate the env,
`
conda activate myenv
`
Then run this
`
python -m ipykernel install --user --name myenv --myenvJupyter "Python myenv"
`
Lastly restart Jupiter and select the environment `myenv` as the python kernel to run.

**Comment by mgbukov** [comment]
This is fixed in quspin 0.3.3
